tests/layer_tests/pytorch_tests/test_view.py

```diff
@@ -0,0 +1,59 @@
+# Copyright (C) 2018-2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import pytest
+from pytorch_layer_test_class import PytorchLayerTest
+import numpy as np
+import torch
+
+
+@pytest.mark.parametrize('input_data', [(np.random.randn(2, 3, 2), np.array(2), np.array(6)),
+                                        (np.random.randn(4), np.array(2), np.array(2))])
+class TestViewListConstruct(PytorchLayerTest):
+
+    def _prepare_input(self):
+        return self.input_data
+
+    def create_model(self):
+        class aten_view_list_construct(torch.nn.Module):
+
+            def forward(self, input_tensor, dim1: int, dim2: int):
+                return input_tensor.view(dim1, dim2)
```

Comment:
From looks of this operatiop it seems like it usually is used with regular ints, not numpy arrays is there a reason for doing it with arrays?